src/components/profile/ProfileNav.tsx

```diff
@@ -14,6 +15,22 @@ export const ProfileNav: React.FC<ProfileNavProps> = ({
   profileData,
   setFilters,
 }) => {
+  useEffect(() => {
+    if (params?.page === "favorites") {
+      setFilters([
```

Comment:
We can use 'replace' as the mergeBehavior in setFilters. Check out https://refine.dev/docs/api-reference/core/hooks/useTable/#filtersdefaultbehavior for more information.